src/components/20-molecules/file-upload/index.js

```diff
@@ -78,6 +78,8 @@ class AXAFileUpload extends LitElement {
         defaultValue:
           'Your file does not correspond with our allowed file-types',
       },
+      onFileDrop: { type: Function, attribute: false },
+      onFileRemove: { type: Function, attribute: false },
```

Comment:
`If attribute is false, the converter, reflect and type options are ignored.`
https://lit-element.polymer-project.org/guide/properties